docker/Dockerfile.archlinux

```diff
@@ -0,0 +1,12 @@
+FROM archlinux AS stage0
+COPY ./products/* /pkgx/
+RUN install -m 755 /pkgx/$(uname -m) /usr/local/bin/pkgx
+RUN install -m 755 /pkgx/pkgm /usr/local/bin/pkgm
+
+FROM archlinux AS stage1
+COPY --from=stage0 /usr/local/bin/pkgx /usr/local/bin/pkgx
+COPY --from=stage0 /usr/local/bin/pkgm /usr/local/bin/pkgm
+
+SHELL ["/bin/bash", "-c"]
+CMD ["/bin/bash", "-i"]
```

Comment:
`-i` should not be necessary as there's nothing useful in bashrc at this point. For example, `PATH` changes should be done at the image level.

`["/bin/bash"]` should be enough.